CRM/Price/Form/Option.php

```diff
@@ -79,10 +79,9 @@ public function setDefaultValues() {
       }
     }
 
-    $memberComponentId = CRM_Core_Component::getComponentID('CiviMember');
     $extendComponentId = CRM_Core_DAO::getFieldValue('CRM_Price_DAO_PriceSet', $this->_sid, 'extends', 'id');
 
-    if (!isset($defaults['membership_num_terms']) && $memberComponentId == $extendComponentId) {
+    if (!isset($defaults['membership_num_terms']) && $this->isComponentPriceOption($memberComponentId, 'CiviMember')) {
```

Comment:
The `$memberComponentId` var is removed just above?

Reply:
right it should be `$extendComponentId` - athought I'd probably move that look up into the function

Reply:
Fixed thanks @demeritcowboy